packages/svelte/src/ambient.d.ts

```diff
@@ -125,6 +125,27 @@ declare namespace $state {
 	 */
 	export function frozen<T>(initial: T): Readonly<T>;
 	export function frozen<T>(): Readonly<T> | undefined;
+
+	/**
+	 * Declares reactive state that is linked to another value. Local mutations will override the linked value
```

Comment:
The word "mutations" sounds like `local_text.foo = bar` will not propagate back up to the original value. I think "reassignments" is a better word (and we likely need to explain this nuance in the `.md` docs)